frontend/jsontablemodel.cpp

```diff
@@ -35,6 +35,9 @@ JsonTableModel::Header JsonTableModel::getHeader() { return m_header; }
 
 QVariant JsonTableModel::headerData(int section, Qt::Orientation orientation, int role) const
 {
+    if (role == 32)
```

Comment:
Don't use a magic number, create your own enum for custom roles.  Also, 32 seems a bit low - I thought most custom roles started at 1000?